iot_center/iot_center.yml

```diff
@@ -0,0 +1,253 @@
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: spectacular-yalow-bfb001
+spec:
+    color: '#be2ee4'
+    name: IoT
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Variable
+metadata:
+    name: zealous-williams-ffb001
+spec:
+    language: flux
+    name: IoT_Device
+    query: "import \"influxdata/influxdb/v1\"\r\nv1.tagValues(bucket: \"iot_center\",
```

Comment:
Since you have a hard coded bucket name of "iot_center", your template should include a Bucket by that name.